### PR TITLE
QA review

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: src/_pages/ProjectDetails/Rewards/Budget/Amount.tsx [hunk 1]
```diff
@@ -8,7 +8,20 @@ type Amount = {
...
 
-  if (budget.currency === Currency.STRK || budget.usdEquivalent === null) {
+  if (!budget.usdEquivalent && !!budget.amount) {
+    return (
+      <>
+        {`${formatMoneyAmount({
+          amount: budget.amount,
+          currency: budget.currency,
+          showCurrency: false,
+        })}`}
```
Comment (haydencleary) [inline src/_pages/ProjectDetails/Rewards/Budget/Amount.tsx:18]:
```suggestion
      {formatMoneyAmount({
          amount: budget.amount,
          currency: budget.currency,
          showCurrency: false,
        })}
```
No?

File: src/_pages/ProjectDetails/Rewards/Budget/Budget.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ export function Budget({
 
   return (
     <>
-      <div className="grid w-full gap-4 md:grid-cols-2 lg:grid-cols-4">
+      <div className="grid w-full gap-4 max-[1470px]:grid-cols-2 max-[700px]:grid-cols-1 min-[1470px]:grid-cols-4">
```
Comment (haydencleary) [inline src/_pages/ProjectDetails/Rewards/Budget/Budget.tsx:45]: Maybe we should think about custom media query tokens ? Not in this PR, just something to keep in mind.

File: src/_pages/ProjectDetails/Rewards/Budget/BudgetCard.tsx [hunk 3]
```diff
@@ -90,6 +90,7 @@ export function BudgetCard({
       className={cn("group px-4 py-5 transition-all lg:px-4 lg:py-5", {
         "relative z-[1] overflow-hidden": type === CardTypes.Remaining,
         "cursor-pointer": !!onClick,
+        "hover:scale-[0.99] hover:opacity-90": !!onClick,
```
Comment (haydencleary) [inline src/_pages/ProjectDetails/Rewards/Budget/BudgetCard.tsx:93]: You can merge these conditions

File: src/_pages/ProjectDetails/Rewards/Budget/Panel.tsx [hunk 4]
```diff
@@ -101,20 +115,20 @@ export function BudgetPanel({ open, close, onPanelChange, projectId }: BudgetPan
...
-          <div className="mt-8 flex flex-col gap-4 border-b-1 border-b-card-border-light pb-6">
-            {values.map(value => (
+          <Tabs tabs={tabs} variant="blue" showMobile mobileTitle={T("project.details.edit.title")} border={true} />
+          <div className="mt-4 flex flex-col gap-4 border-b-1 border-b-card-border-light pb-6">
+            {sortedByDollarsEquivalent.map(value => (
               <Card
                 key={value.currency}
                 border="light"
-                className="flex flex-row items-center justify-between bg-card-background-light"
+                className="flex flex-row items-center justify-between bg-card-background-light p-4 lg:p-4"
```
Comment (haydencleary) [inline src/_pages/ProjectDetails/Rewards/Budget/Panel.tsx:124]: Won't one padding class do ? If it is to override the default media query maybe `!p-4` would be better.

File: src/_pages/ProjectDetails/Rewards/Budget/Panel.tsx [hunk 5]
```diff
@@ -136,8 +150,11 @@ export function BudgetPanel({ open, close, onPanelChange, projectId }: BudgetPan
               </Card>
             ))}
           </div>
-          <div className="pt-6">
-            <Card border="light" className="od-bg-budget flex flex-row items-center justify-between border-0">
+          <div className="py-6">
+            <Card
+              border="light"
+              className="od-bg-budget flex flex-row items-center justify-between border-0 p-4 lg:p-4"
```
Comment (haydencleary) [inline src/_pages/ProjectDetails/Rewards/Budget/Panel.tsx:156]: Same remark here for the padding

Comment (haydencleary) [inline src/_pages/ProjectDetails/Rewards/Budget/Panel.tsx:156]: `flex-row` isn't necessary here, in tailwind `flex` is row by default.

**Review by haydencleary — CHANGES_REQUESTED** [review]

File: src/_pages/ProjectDetails/Rewards/Budget/Panel.tsx (hunk 4 above)
Comment (alexbeno) [reply src/_pages/ProjectDetails/Rewards/Budget/Panel.tsx:124]: yes maybe important could be great but if we want to add another breakpoint i we will need to add important everywhere ?

File: src/_pages/ProjectDetails/Rewards/Budget/Budget.tsx (hunk 2 above)
Comment (alexbeno) [reply src/_pages/ProjectDetails/Rewards/Budget/Budget.tsx:45]: yes would be greate we have many breakpoint that was not handle by the tailwind breakpoint

File: src/_pages/ProjectDetails/Rewards/Budget/Panel.tsx (hunk 5 above)
Comment (alexbeno) [reply src/_pages/ProjectDetails/Rewards/Budget/Panel.tsx:156]: same response, waiting for your answers

**Review by haydencleary — APPROVED** [review]

**Pull request merged**